packages/thunk/package.json

```diff
@@ -15,6 +15,7 @@
   ],
   "dependencies": {
     "@redux-tools/namespaces": "^0.8.2",
+    "@redux-tools/namespaces-react": "^0.8.2",
```

Comment:
thunk is platform agnostic, so it cannot possibly depend on namespaces-react, please remove :)